modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,694 @@
+    data-mce-annotation="tinycomments">
+      Comment
+  </span>
+</p>
+----
+
+— {productname} 6.3 will automatically expand the selection to include the whole word, ‘Comment’.
+
+=== New `color_map_foreground` and `color_map_background` options store custom colors selected using the Text color and Background color menus.
+
+{productname} 6.3 has two new and separate color palettes — `color_map_foreground` and `color_map_background` — for custom Text colors and custom Background colors.
+
+When custom colors are selected from the Text color or Background color menus, these colors are now stored in these separate color palettes, `color_map_foreground` and `color_map_background`, respectively.
+
+When chosen, these custom colors are still added to the Text color and Background color menus. But they are now stored separately from the default color sets used by both menus.
+
+NOTE: When a user adds a custom color to the `color_map_background` palette, the editor does not replicate the new color in the `color_map_foreground` palette.
+
+=== Added optional storageKey property to `colorinput` component
+
+{productname} 6.3 introduced two new color palettes for `color_map_background` and `color_map_foreground`.
+
+This addition introduced changes to the `colorcache`: it is no longer considered a singleton. The `colorcache` can now be accessed with the help of a custom ID, which can now create custom `color_map` colors, selected by the user to be assigned to either the `backcolor` or `forecolor` palettes separately.
+
+To accompany this, {productname} 6.3 now allows for the creation of a separate `storageKey` property for the pre-provided colors in `color_map_forground` and `color_map_background` respectively. `storageKey` properties can also be set to `color_cols`, the `colorinput` component, and `colorswatch`.
+
+NOTE: If no separate values are set for `color_map_forground` and `color_map_background`, they still default to the shared or previous values as before.
+
+
+=== A new addView API was added to editor.ui.registry 
+
+The new `addView` API registers a new view container.
+
+This container, which is off (hidden) by default, is attached to the main view.
+
+In this default state, the main editor view is shown and the `addView`-specific view is hidden.
+
+When `addView` is turned on, the main editor view is hidden, and the `addView`-specific view is shown.
+
+Accompanying this new API is a new `ToggleView` command.
+
+The `ToggleView` command can, as its name suggests, toggle view visibility.
+
+As well, the `ToggleView` command can be queried for its current state.
+
+
+=== New color_default_foreground and color_default_background options to set the initial default color for the forecolor and backcolor toolbar buttons and menu items.
+
+{productname} 6.3 adds a new feature that allows the default color for the `color_default_foreground` and `color_default_background` in the toolbar button menu to be pre-defined.
```

Comment:
```suggestion
{productname} 6.3 adds a new feature that allows the default color for the icons in the foreground color and background color toolbar buttons to be pre-defined.
```
The default colors in the forecolor and backcolor icons can now be defined. Saying "the `color_default_foreground` toolbar button menu default color can now be defined" sounds redundant and doesn't make sense as they are options not toolbar buttons.